tests/utils/domainService.test.js

```
@@ -1,23 +1,26 @@
+    const validAddress =
+      "0x061b6c0a78f9edf13cea17b50719f3344533fadd470b8cb29c2b4318014f52d3";
     const expectedDomain = "fricoben.stark";
 
     // Mocking the fetch function to return a predefined response
     const mockResponse = { domain: expectedDomain };
     const fetchMock = jest.fn().mockResolvedValue({
-        json: jest.fn().mockResolvedValue(mockResponse),
+      json: jest.fn().mockResolvedValue(mockResponse),
     });
-    (fetch as jest.Mock) = fetchMock;
+    fetch = fetchMock;
```

Biome (1.9.4)

[error] A global variable should not be reassigned.

Assigning to a global variable can override essential functionality.

(lint/suspicious/noGlobalAssign)